projects/03/a/PC.hdl

```diff
@@ -16,5 +16,11 @@ CHIP PC {
     OUT out[16];
 
     PARTS:
-    // Put your code here:
+    Inc16(in=outt, out=inc16);
+
+    Mux16(a=outt, b=inc16, sel=inc, out=incorelse);
+    Mux16(a=incorelse, b=in, sel=load, out=loaded);
+    Mux16(a=loaded, b=false, sel=reset, out=last);
+
+    Register(in=last, load=true, out=outt, out=out);

```

Comment:
This is absolutely correct, well done! 🎉 

The computer doesn’t care of course, but for practice purposes, could we think of more systematic names for these internal pins? It’s a little hard to understand the difference between `out` and `outt`, for example, and `incorelse`/`loaded`/`last` is a bit all over the place. I don’t have any brilliant ideas here but maybe if we think about it we can come up with something.

Reply:
Fixed this in the latest commit!